src/player.h

```diff
@@ -1154,6 +1154,8 @@ class Player final : public Creature, public Cylinder
 		void forgetInstantSpell(const std::string& spellName);
 		bool hasLearnedInstantSpell(const std::string& spellName) const;
 
+		void updateRegeneration();
```

Comment:
Could be moved to private scope I guess 🤔

Reply:
Good idea

Reply:
https://twitter.com/i/status/1430322259583983621

Reply:
Could be moved to cpp only and not be in the header